en/sql/function/json_fn.rst

```diff
@@ -477,3 +477,115 @@ The following json path will match all json paths that end with a json array ind
       '2'
       '3'
       NULL
+
+JSON_CONTAINS_PATH
+===================================
+
+.. function:: JSON_CONTAINS_PATH (json_doc, one/all, json path [, json path] ...)
+
+  The **JSON_CONTAINS_PATH** function verifies whether the the given paths exist inside the json_doc.
+  1 is returned with one/all argument given as 'all' only if all given paths exist inside the json_doc, 0 is returned otherwise. In case one/all argument is 'one', 1 is returned if at least one json path exists inside the json_doc, 0 otherwise.
```

Comment:
Please rephrase:
```suggestion
  When one/all argument is 'all', all given paths must exist to return 1. Returns 0 otherwise.
  When one/all argument is 'one', it returns 1 if any given path exists. Returns 0 otherwise.
```